cocotb/share/lib/fli/FliImpl.cpp

```diff
@@ -434,6 +434,31 @@ void FliImpl::get_sim_precision(int32_t *precision)
     *precision = mti_GetResolutionLimit();
 }
 
+const char *FliImpl::get_simulator_product()
+{
+    if (m_product.empty() && m_version.empty()) {
+        const char *c_info       = mti_GetProductVersion();      // Returned pointer must not be freed
+        const std::string info   = c_info;
+        const std::string search = " Version ";
+        const std::size_t found  = info.find(search);
+
+        if (found != std::string::npos) {
+            m_product = info.substr(0, found);
+            m_version = info.substr(found+search.length());
+        } else {
+            m_product = "UNKNOWN";
```

Comment:
```suggestion
            m_product = info;
```
Better to assume we have the product string than drop it on the ground